docs/_posts/gadde5300/2021-12-14-distilbert_uncased_te.md

```diff
@@ -0,0 +1,122 @@
+task: Embeddings
+language: te
+edition: Spark NLP 3.1.0
+spark_version: 3.0
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This is a DistilBERT language model pre-trained on ~2 GB of the monolingual training corpus. The pre-training data was majorly taken from OSCAR. This model can be fine-tuned on various downstream tasks like text classification, POS-tagging, question-answering, etc. Embeddings from this model can also be used for feature-based training.
+
+## Predicted Entities
+
+
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+<button class="button button-orange" disabled>Open in Colab</button>
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/public/models/distilbert_uncased_te_3.1.0_3.0_1639472349482.zip){:.button.button-orange.button-orange-trans.arr.button-icon}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+document_assembler = DocumentAssembler()\
+    .setInputCol('text')\
+    .setOutputCol('document')
+
+sentence_detector = SentenceDetector() \
+    .setInputCols(['document'])\
+    .setOutputCol('sentence')
+
+tokenizer = Tokenizer()\
+    .setInputCols(['sentence']) \
+    .setOutputCol('token')
+
+distilbert_loaded = DistilBertEmbeddings.pretrained("distilbert_uncased", "te"))\
+  .setInputCols(["sentence",'token'])\
+  .setOutputCol("embeddings")\
+  .setCaseSensitive(False)
+
+nlp_pipeline = Pipeline(stages=[document_assembler, sentence_detector, tokenizer, distilbert_loaded])
+
+pipeline_model = nlp_pipeline.fit(spark.createDataFrame([[""]]).toDF("text"))
+
+result = pipeline_model.transform(spark.createDataFrame([['ఆంగ్ల పఠన పేరాల యొక్క గొప్ప మూలం కోసం చూస్తున్నారా? మీరు సరైన స్థలానికి వచ్చారు.']], ["text"]))
+
+```
+```scala
+val document_assembler = DocumentAssembler() 
+    .setInputCol("text") 
+    .setOutputCol("document")
+
+val sentence_detector  = SentenceDetector()
+    .setInputCols("document")
+    .setOutputCol("sentence")
+
+val tokenizer = Tokenizer()\
+    .setInputCols("sentence") \
+    .setOutputCol("token")
+
+val distilbert_loaded = DistilBertEmbeddings.pretrained("distilbert_uncased", "te"))\
+  .setInputCols("sentence","token")\
+  .setOutputCol("embeddings")\
+  .setCaseSensitive(False)
+
+val nlp_pipeline = new Pipeline().setStages(Array(document_assembler, sentence_detector, tokenizer, distilbert_loaded))
+
+val data = Seq("ఆంగ్ల పఠన పేరాల యొక్క గొప్ప మూలం కోసం చూస్తున్నారా? మీరు సరైన స్థలానికి వచ్చారు.").toDF("text")
+
+val result = pipeline.fit(data).transform(data)
+```
+</div>
+
+## Results
+
+```bash
+| Token    	| Embeddings                                                           	|
+|----------	|----------------------------------------------------------------------	|
+| ఆంగ్ల     	| [1.3526772, 0.66792506, 0.80145407, -1.7625582, 1.222954, ...]        	|
```

Comment:
Can you fix the space for the last vertical bar `|`?

Reply:
@gadde5300 
I fixed the issue here.